client/src/Views/Home/Campaigns/Campaign/CampaignHeader.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Grid, Typography, makeStyles } from "@material-ui/core";
+import { withRouter } from "react-router-dom";
+
+import StyledButton from "Components/Button/StyledButton";
+
+const useStyles = makeStyles({
+  root: {
+    width: "100%",
+    padding: "2rem 3rem"
+  },
+  title: {
+    flexGrow: 1
+  },
+  name: {
+    fontSize: "2rem",
+    fontWeight: "bold"
+  }
+});
+
+const CampaignHeader = props => {
+  const classes = useStyles();
+
+  const handleAddProspects = () => {
+    props.history.push(`/home/prospects`);
+  };
+
+  const handleAddStep = () => {
+    props.history.push(`${props.match.url}/step`);
+  };
```

Comment:
Is this component being used anywhere? Currently there are only two pages (Summary and Prospects) for the individual campaign detail and both of them already have header components. Also, opening the editor should no longer be done by changing the browser location directly.

Reply:
Yes, also probably due to not merging dev correctly.